pegasus/sites.v3/hourofcode.com/views/census_questions.haml

```diff
@@ -0,0 +1,56 @@
+:ruby
+  how_many_questions = [
+    {
+      id: "hoc-how-much",
+      name: "hoc_s",
+      label: hoc_s(:census_how_many_hoc)
+    },
+    {
+      id: "after-school-how-much",
+      name: "after_school_s",
+      label: hoc_s(:census_how_many_after_school)
+    },
+    {
+      id: "ten-hour-how-much",
+      name: "ten_hours_s",
+      label: hoc_s(:census_how_many_ten_hours)
+    },
+    {
+      id: "twenty-hour-how-much",
+      name: "twenty_hours_s",
+      label: hoc_s(:census_how_many_twenty_hours)
+    }
+  ]
+
+.form-group
+  %label.control-label=hoc_s(:census_connection)
+  %select#role.form-control{name: "role_s", type: "select"}
+    %option{value: "", selected: true, disabled: true}
+    %option{value: "teacher"}=hoc_s(:teacher)
+    %option{value: "administrator"}=hoc_s(:administrator)
+    %option{value: "parent"}=hoc_s(:parent)
+    %option{value: "volunteer"}=hoc_s(:volunteer)
```

Comment:
apologies if I've caught you between changes, but it looks like you'll have to change it to `volunteer_or_advocate` on this line too